python/ray/data/dataset.py

```diff
@@ -210,9 +210,9 @@ class Datastream(Generic[T]):
         >>> import ray
         >>> ds = ray.data.range(1000)
         >>> # Transform in parallel with map_batches().
-        >>> ds.map_batches(lambda batch: [v * 2 for v in batch])
+        >>> ds.map_batches(lambda batch: [v * 2 for v in batch]) # doctest: +ELLIPSIS
```

Comment:
I don't think you need to add `# doctest: +ELLIPSIS`, if you set `num_blocks=...` below.

Reply:
I think the `ELLIPSIS` option _required_ for python to interpret the ellipsis as matching any string - see https://docs.python.org/3/library/doctest.html#doctest.ELLIPSIS.

Reply:
hmm I don't think it's required for Ray here, we have several places with `num_blocks=...` already - https://github.com/ray-project/ray/blob/master/python/ray/data/dataset.py#L222-L230 .